.github/workflows/pythonpackage.yml

```diff
@@ -24,22 +25,17 @@ jobs:
         pip install -r requirements-dev.txt
     - name: linter (codespell)
       run: pre-commit run --all-files
-      shell: bash
```

Comment:
Why not be explicit here? I know that on almost all systems bash is the default, but there are some where it isn't, leading to hard to debug issues, when bashisms are used in the called scripts - I would be okay, but then please put some reasoning into the commit message

Reply:
Thanks for your review.

The reason was to remove unneeded code from each step. However, it's possible to set a global default instead, so the shell remains explicit but isn't repeated in each step. Should I change the PR in that direction?

Reply:
Yes, please - if we will have a knob to adjust it globally I'm fine with your suggestion

Reply:
LGTM